config/manifests/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ resources:
 - ../default
 - ../samples
 - ../scorecard
+- ../openshift
```

Comment:
Looks like you still need to create `config/openshift/kustomization.yaml` with this content:
> 
> ```yaml
> resources:
> - cryostat-quickstart.yaml
> ```